Makefile

```diff
@@ -89,7 +89,6 @@ src/Java.Runtime.Environment/Java.Runtime.Environment.dll.config: src/Java.Runti
 	sed -e 's#@JI_JVM_PATH@#$(JI_JVM_PATH)#g' -e 's#@OS_NAME@#$(DLLMAP_OS_NAME)#g' -e $(JAVA_RUNTIME_ENVIRONMENT_DLLMAP_OVERRIDE_CMD) < $< > $@
 
 fxcop: lib/gendarme-2.10/gendarme.exe bin/GendarmeDebug/Java.Interop.dll
-	cp src/Java.Interop/obj/Gendarme/Java.Interop.dll.mdb bin/GendarmeDebug/
```

Comment:
We still need to copy the mdb file to have lines information in the gendarme output.

Reply:
@radekdoulik there isn't an `mdb` in here at all now. Maybe a change in the latest mono?

`bin/GendarmeDebug/` already has the pdb: `Java.Interop.pdb`. So is this line not needed anymore?

Reply:
There is an mdb file, check https://jenkins.mono-project.com/job/Java.Interop-pr-builder/ws/src/Java.Interop/obj/Gendarme/

The Gendarme itself is old, and supports only mdb symbol files, thus we use it here until the Gendarme is updated.